### ISSUE TITLE
GetOptionsWithHeaders

### ISSUE THREAD

**Issue by wvdvegt** [inthehand/InTheHand.Forms#49]
I have an issue on iOS with the latest stable version (and an older one as well).

If i specify a source like

` Media.Source = new Uri("https://<host>/<path>.mp4");`

I end up with an exception in iOS that 'Value cannot be null). 

More specific error location:

```
Value cannot be null.
Parameter name: url
  at AVFoundation.AVUrlAsset.FromUrl (Foundation.NSUrl url, Foundation.NSDictionary options)
```

**Comment by wvdvegt** [comment]
Title might be a bit off.

**Comment by wvdvegt** [comment]
Think i got the cause:

 **the iOS renderer does not seem to like spaces in the filename being part of the URL.** 

I tried with both spaces and %20, both failed. Noticed it was reported at the end of #10

**Comment by peterfoot** [comment]
Fixed in 1.0.2018.828
